CustomPermissionClaimHandler/README.md

```diff
@@ -0,0 +1,47 @@
+# CustomPermissionClaimHandler
```

Comment:
Would be great if we can follow the given readme structure.

  ## Project name.
  ### Introduction / Use case.
  Please explain what your sample does and the use case. Use external links, images and diagrams if possible.
  
  ### Applicable product versions.
  List all WSO2 IS versions that can be used with this sample.
  
  ### How to use.
  Please provide detailed steps for using the sample. Include steps to building, patching, configurations that need to do, etc.
  
  ### Testing the project.
  Please provide steps for testing the sample. Provide example inputs and outputs as well.